lib/instrumentation/modules/cassandra-driver.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+/**
+ * TODO: Support batch queries
+ */
+
+var semver = require('semver')
+var sqlSummary = require('sql-summary')
+
+var shimmer = require('../shimmer')
+
+module.exports = function (cassandra, agent, version, enabled) {
+  if (!enabled) return cassandra
+  if (!semver.satisfies(version, '^3.0.0')) {
+    agent.logger.debug('cassandra version %s not supported - aborting...', version)
+    return cassandra
+  }
+
+  agent.logger.debug('shimming cassandra.Client.prototype._innerExecute')
+  shimmer.wrap(cassandra.Client.prototype, '_innerExecute', wrapInnerExecute)
+
+  return cassandra
+
+  function wrapInnerExecute (original) {
+    return function wrappedInnerExecute (...args) {
```

Comment:
The spread operator isn't isn't supported in Node 4 without a flag.

We can drop support for v4, but should bump major if we do. If not too much of a hassle I'd like to wait bumping major until v2 of the intake API. What do you think?

Reply:
Yeah, I don't need to use the spread operator. I just keep forgetting it didn't exist yet in 4.x. 😅

Reply:
fixed.